public/app/features/dashboard/components/ShareModal/SharePublicDashboard/SharePublicDashboardUtils.ts

```diff
@@ -13,6 +13,7 @@ export interface PublicDashboard {
   uid: string;
   dashboardUid: string;
   timeSettings?: object;
+  timeSelectionEnabled: boolean;
```

Comment:
looks like there is a timeSettings object here already?

Reply:
Yes, currently we aren't using that property which will be in the future the json object with all the time settings.